Readme.md

```diff
@@ -54,11 +54,11 @@ pino(transport)
 
 ### Events
 
-| Name    | Callback Signature               | Description                                                                                                                                          |
-|---------|----------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `open`  | `(address: AddressInfo) => void` | Emitted when the TCP or UDP connection is established.                                                                                               |
-| `error` | `(error: Error) => void`         | Emitted when an error occurs on the TCP or UDP socket.                                                                                               |
-| `close` | `(hadError: Boolean) => void`    | Emitted after the TCP or UDP socket is closed. The argument `hadError` is a boolean which says if the socket was closed due to a transmission error. |
+| Name          | Callback Signature               | Description                                                                                                                                          |
+|---------------|----------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `open`        | `(address: AddressInfo) => void` | Emitted when the TCP or UDP connection is established.                                                                                               |
+| `socketError` | `(error: Error) => void`         | Emitted when an error occurs on the TCP or UDP socket. The socket won't be closed.                                                                   |
+| `close`       | `(hadError: Boolean) => void`    | Emitted after the TCP or UDP socket is closed. The argument `hadError` is a boolean which says if the socket was closed due to a transmission error. |
```

Comment:
Please document how to get the previous behavior

Reply:
Done